apps/prairielearn/public/stylesheets/local.css

```diff
@@ -310,3 +310,132 @@ small .user-output-invalid {
   margin-left: 5px;
   background: #fff;
 }
+
+/***********************************
+ * Button focus accessibility styles
+ ***********************************/
+
+/**
+ * TODO: After we're running with only Bootstrap 5, these can be collapsed into
+ * a single rule with all the different selectors, since they can all share the
+ * same var(--bs-btn-hover-color) value.
+ */
+
+.btn.btn-primary:focus:focus-visible {
+  outline-style: solid !important;
+  outline-color: var(--bs-btn-hover-color, #ffffff) !important;
```

Comment:
This offers forwards compatibility with Bootstrap 5, which uses slightly different colors. In BS4, this variable isn't defined, so the fallback will be used instead.